source/collimation.f90

```diff
@@ -2588,38 +2343,31 @@ subroutine collimate_do_collimator(stracki)
       write(outlun,*) 'RMS error on halfgap [sigma]:  ', gap_rms_error(icoll)
       write(outlun,*) ' '
 
-      write(collgaps_unit,'(i10,1x,a,4(1x,e19.10),1x,a,6(1x,e13.5))')   &
-     &icoll,db_name1(icoll)(1:12),                                      &
-     &db_rotation(icoll),                                               &
-     &tbetax(ie), tbetay(ie), calc_aperture,                            &
-     &db_material(icoll),                                               &
-     &db_length(icoll),                                                 &
-     &sqrt(tbetax(ie)*myemitx0_collgap),                                &
-     &sqrt(tbetay(ie)*myemity0_collgap),                                &
-     &db_tilt(icoll,1),                                                 &
-     &db_tilt(icoll,2),                                                 &
-     &nsig
+      write(collgaps_unit,"(i4,1x,a16,4(1x,e19.10),1x,a4,5(1x,e13.5),1x,f13.6)") &
+        icoll,cdb_cName(icoll)(1:16),cdb_cRotation(icoll),tbetax(ie),tbetay(ie),calc_aperture, &
+        cdb_cMaterial(icoll),cdb_cLength(icoll),sqrt(tbetax(ie)*myemitx0_collgap), &
+        sqrt(tbetay(ie)*myemity0_collgap),db_tilt(icoll,1),db_tilt(icoll,2),nsig
```

Comment:
I would probably also extract what concerns `collgaps` into a separate subroutine in `coll_db`